client/client.go

```diff
@@ -121,13 +121,13 @@ type response struct {
 }
 
 // errorResult is the real value of response.Result when an error occurs.
-// Note that only the 'Str' field is unmarshaled from JSON representation.
+// Note that only the 'message' field is unmarshaled from JSON representation.
 type errorResult struct {
-	Str string `json:"str"`
+	Message string `json:"message"`
```

Comment:
None beyond that we aren't using it for anything yet in the client itself. When the client supports installing things, including the license agreement thing, it'll have to grow the right bits.